daemon/indexing/listener/listener.js

```diff
@@ -11,26 +11,18 @@ const db = require('../lib//db.js')
 // ---------------
 
 // Todo
-// - Allow configuring web3 endpoint, network, and IPFS endpoint
-// - When catching up, work in smaller batches
 // - Handle blockchain splits/winners
-// - Possibly switch lastLogBlock to be closer to last found block
 // - Include current-as-of block numbers in POSTs
 // - Persist starting point in DB
-// - Perhaps send related data as it was at the time of the event, not crawl time
+// - Perhaps send related data as it was at the time of the event, not as of crawl time
 // - Possible configurable log levels
 
-const web3Provider = new Web3.providers.HttpProvider('http://localhost:8545')
-const web3 = new Web3(web3Provider)
-const o = new Origin({
-  ipfsDomain: 'origin-js',
-  ipfsGatewayProtocol: 'http',
-  ipfsGatewayPort: 8080,
-  web3
-})
+let web3
+let o
 
 MAX_RETRYS = 10
 MAX_RETRY_WAIT_MS = 2 * 60 * 1000
+MAX_BATCH_BLOCKS = 30000
```

Comment:
Maybe it would make sense to put all this into `.env` file. As @franckc mentions depending on the volume of listings on our contracts we might need to tweak these numbers. At it is nice we can do that without the necessary code changes.

Reply:
Actually this might change between runs of the listener perhaps a better way would be to write the default values in .env file, an make it overridable with cmd parameters.

Reply:
I've dropped the max_batch_blocks down to 3000 (half a day) and added a note.